### PR TITLE
add besu goerli to stakersUI

### PR DESCRIPTION
**Pull request by alexpeterson91**
Adding Besu-Goerli to Stakers

### CODE REVIEW

File: packages/dappmanager/src/modules/stakerConfig/utils.ts [hunk 1]
```diff
@@ -95,7 +95,8 @@ export function getNetworkStakerPkgs(network: Network): {
           {
             dnpName: "goerli-nethermind.dnp.dappnode.eth",
             minVersion: "1.0.1"
-          }
+          },
+          { dnpName: "besu-goerli.dnp.dappnode.eth", minVersion: "0.1.0" }
```
Comment (pablomendezroyo) [inline packages/dappmanager/src/modules/stakerConfig/utils.ts:99]: Could you change the dnp name and repo to be consistent with the naming of the other goerli exec clients (goerli-geth and goerli-erigon) from `besu-goerli` to `goerli-besu`

**Review by pablomendezroyo — CHANGES_REQUESTED** [review]

File: packages/dappmanager/src/modules/stakerConfig/utils.ts (hunk 1 above)
Comment (alexpeterson91) [reply packages/dappmanager/src/modules/stakerConfig/utils.ts:99]: Yes will do.

Comment (alexpeterson91) [reply packages/dappmanager/src/modules/stakerConfig/utils.ts:99]: @Nabsku can yiu do this to the package and the 5 CLs? I'll fix here. And we'll pick up in a few more hours

**Comment by alexpeterson91** [comment]
goerli Besu must be published before merging,

**Comment by Nabsku** [comment]
changed all CLs to `goerli-besu` instead of `besu-goerli`!

**Review by alexpeterson91 — COMMENTED** [review]
I'm gonna put together hashes of the packages needed for this all to work properly there's a pr for each CL and the W3S. And since adding clients to the stakersUI now requires all of this lodestar prater is good to go. Lion signed off on it. So we may want to add them both to avoid having to do multiple core and all client updates. Anyways I'll have hashes to test both of the clients posted this week.

**Review by 3alpha — APPROVED** [review]

**Comment by pablomendezroyo** [comment]
Whats the current status of this?

**Review by pablomendezroyo — APPROVED** [review]

**Pull request merged**